docs/modules/druid/pages/usage-guide/discovery.adoc

```diff
@@ -3,6 +3,7 @@
 :routerPort: 8888
 
 = Discovery
+:page-aliases: discovery.adoc
```

Comment:
I think this page is missing in the index.

Reply:
Oh damn! Good catch, thx. I've added it, it's there now: https://deploy-preview-371--stackable-docs.netlify.app/home/nightly/druid/usage-guide/discovery.html